### DISCUSSION TITLE
iframe tag embedded in the ketcher how to hide some buttons

### DISCUSSION THREAD

**Discussion by niuzhenjiang** [epam/ketcher#1219] [Q&A] [answered]
how can i do it,thanks

**Comment by AndreiMazol** [comment, marked as answer]
Hi @niuzhenjiang ,

controls can be hidden in both modes (remote and standalone) by passing 'hiddenControls' query parameter, for example to hide 'open' and 'save' buttons it is necessary to pass the following values:

http://localhost:5000/?hiddenControls=open,clear

unfortunately this functionality is not documented so the list of all available options can be found in https://github.com/epam/ketcher/blob/master/packages/ketcher-react/src/script/ui/action/index.js (look at use of 'isHidden' function)

Regards,
Andrei

**Reply by bhans-uvic** [reply]
I am trying to achieve the same result while using Editor react component. I have tried passing prop buttons={{open:{hidden:true}}} to the Editor but it doesn't work since open is not an accepted value on ButtonsConfig. How can I hide open, copy, etc while using the Editor?
